fluent-bundle/Cargo.toml

```diff
@@ -19,7 +19,7 @@ categories = ["localization", "internationalization"]
 
 [dependencies]
 fluent-locale = "^0.4.1"
-fluent-syntax = "^0.8"
+fluent-syntax = { path = "../fluent-syntax/" }
```

Comment:
Are you adding this just for testing? Or do you plan to keep this after this implementation is published?

Reply:
definitely will switch back after releasing fluent-syntax 0.9 and before releasing fluent-bundle 0.6 :)